src/stylesheets/animations.less

```diff
@@ -170,11 +210,24 @@
 
 .sk-init {
     bottom: @widget-close-bottom-md;
-    @media (min-width: @screen-lg-min) {
+
+    @media (min-width: @screen-lg-min) and (min-height: @screen-md-ht-max) {
         bottom: @widget-close-bottom-lg;
     }
 
-    @media (max-width: @screen-sm-min) {
+    @media (min-height: 488px) and (max-height: @screen-md-ht-max) and (min-width: @screen-sm-min) {
+        bottom: @widget-close-bottom-md;
+    }
+
+    @media (max-height: 488px) {
+        bottom: @widget-close-bottom-fs;
+    }
+
+    @media (max-height: @screen-sm-ht-max) {
         bottom: @widget-close-bottom-sm;
     }
+
+    @media (max-width: @screen-sm-min) {
+        bottom: @widget-close-bottom-fs;
+    }
 }
```

Comment:
When `max-height: @screen-sm-ht-max and min-width: @screen-lg-min` the widget is hidden